starters/vue3-xstate-css/.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "singleQuote": true,
+  "semi": true,
+  "tabWidth": 2,
+  "trailingComma": "es5",
+  "useTabs": false
```

Comment:
it looks like we might need to change this to "useTabs": true because in the spec it says this


It has been proven that tabs are better than spaces for accessibility purposes. As such, all tooling should be configured to use tabs when possible. YAML is a known exception.